examples/seismic/tti/wavesolver.py

```diff
@@ -171,16 +160,8 @@ def adjoint(self, rec, srca=None, p=None, r=None, vp=None,
             The computed wavefield first component.
         r : TimeFunction, optional
             The computed wavefield second component.
-        vp : Function or float, optional
-            The time-constant velocity.
-        epsilon : Function or float, optional
```

Comment:
Can we leave these in the docstring so that it's easier to know what inputs can be expected.